lib/ardb/adapter/base.rb

```diff
@@ -1,16 +1,21 @@
+require 'ardb'
+
 module Ardb; end
-class Ardb::Adapter
+module Ardb::Adapter
 
   class Base
 
-    attr_reader :ar_connect_hash, :database
+    attr_reader :config, :ar_connect_hash, :database
```

Comment:
@jcredding we discussed renaming `ar_connect_hash` to just `connect_hash` - it is kinda redundant at this level and `ar` seems a little haphazard.  You can do this in a follow up hotfix as well.